clients/callback_client.go

```diff
@@ -181,6 +198,15 @@ func (ts TranscodeStatus) String() string {
 
 // The various status messages we can send
 
+type RecordingEvent struct {
+	When        string `json:"when"`
+	StreamId    string `json:"streamId"`
```

Comment:
Is this the Mist stream name (like `video+123`) or the Livepeer Studio Stream ID. I actually doubt it is the Studio ID since catalyst doesn't know Studio, so in that case WDYT of naming this `stream_name` instead?